Dockerfile

```diff
@@ -19,7 +19,7 @@ ARG GO_BUILD_TAGS
 
 COPY pkg ./pkg
 COPY cmd ./cmd
```

Comment:
The `apt-get` commands can be removed too

Reply:
fixed